### PR TITLE
Added 4 model classes

### PR DESCRIPTION
**Pull request by Maloulou99**
Added a pull request for test

### CODE REVIEW

File: src/main/java/com/example/kinoxpbackend/model/Customer.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.example.kinoxpbackend.model;
+
+public class Customer {
```
Comment (ErdalErdem) [inline src/main/java/com/example/kinoxpbackend/model/Customer.java:25]: test

**Review by ErdalErdem — COMMENTED** [review]
test

File: src/main/java/com/example/kinoxpbackend/model/Customer.java [hunk 2]
```diff
@@ -11,6 +11,12 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
```
Comment (PrinceH00) [inline src/main/java/com/example/kinoxpbackend/model/Customer.java:14]: test

**Review by PrinceH00 — COMMENTED** [review]
done

**Pull request merged**